webservices/common/models/itemized.py

```diff
@@ -44,11 +40,6 @@ class BaseRawItemized(db.Model):
     back_reference_schedule_name = db.Column('br_sname', db.String)
     load_timestamp = db.Column('create_dt', db.DateTime)
 
-
-    @hybrid_property
-    def report_type(self):
-        return self.filing.form_type
-
```

Comment:
Oh wow, I didn't realize the method was defined twice in there, d'oh!  Thanks for fixing!